scripts/esbuild/util.ts

```diff
@@ -109,12 +109,23 @@ export function runBuilds(builds: ESBuildOptions[], opts: BuildOptions): Promise
  * @returns a base set of options
  */
 export function getBaseEsbuildOptions(): ESBuildOptions {
-  return {
+  const options: ESBuildOptions = {
     bundle: true,
     legalComments: 'inline',
     logLevel: 'info',
     target: getEsbuildTargets(),
   };
+
+  // if the `build` subcomment is called with the `DEBUG` env var, like
+  //
+  // DEBUG=true npm run build
+  //
+  // then we should produce sourcemaps.
+  if (process.env.DEBUG) {
+    options.sourcemap = 'linked';
+  }
```

Comment:
Can we document this in `CONTRIBUTING.md` and potentially add a new top level `package.json` script to perform a `npm run build.dev` (or something like that) where we automatically set that env var?

Reply:
oh I meant to comment on this but didn't submit the review! I think I'd lean towards putting some docs in `CONTRIBUTING.md` but `package.json` is already quite crowded imo

Reply:
I added a little note to `CONTRIBUTING.md` in the section on debugging